Apps/Sandcastle/gallery/CZML Polygon - Interpolating References.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
```

Comment:
This example cuts into the globe:

![image](https://cloud.githubusercontent.com/assets/782098/10168956/ec54d612-669c-11e5-88e3-f640e6179617.png)


Can we put it higher or make it smaller?